pkg/cmd/server/admin/create_servercert.go

```diff
@@ -48,8 +48,8 @@ func NewCommandCreateServerCert(commandName string, fullName string, out io.Writ
 	flags := cmd.Flags()
 	BindGetSignerCertOptions(options.GetSignerCertOptions, flags, "")
 
-	flags.StringVar(&options.CertFile, "cert", "openshift.local.certificates/user/cert.crt", "The certificate file.")
-	flags.StringVar(&options.KeyFile, "key", "openshift.local.certificates/user/key.key", "The key file.")
+	flags.StringVar(&options.CertFile, "cert", "openshift.local.config/master/hostname-server.crt", "The certificate file.")
+	flags.StringVar(&options.KeyFile, "key", "openshift.local.config/master/hostname-server.key", "The key file.")
```

Comment:
There are lots of server.crt's for various bits being hosted.  Hinting that they should be qualified seems useful.